crates/re_viewer_context/src/space_view/view_context_system.rs

```diff
@@ -28,8 +29,10 @@ pub trait ViewContextSystem {
     fn as_any(&self) -> &dyn std::any::Any;
 }
 
+// TODO(jleibs): This probably needs a better name now that it includes class name
 pub struct ViewContextCollection {
     pub(crate) systems: HashMap<ViewSystemName, Box<dyn ViewContextSystem>>,
+    pub(crate) space_view_class_name: SpaceViewClassName,
```

Comment:
Given that every view context is always associated with a space view, capturing the name here seemed more straight-forward than creating a custom context for it.